kuksa-client/kuksa_client/cli_backend/ws.py

```diff
@@ -153,18 +160,33 @@ def updateMetaData(self, path, jsonStr, timeout=5):
     def getMetaData(self, path, timeout=5):
         """Get MetaData of the parameter"""
         req = {}
-        req["action"] = "getMetaData"
-        req["path"] = path
+        if self.subprotocol == "VISSv2":
+            req["action"] = "get"
+            req["path"] = path
+            req["filter"] = {"type": "static-metadata"}
+        else:
+            req["action"] = "getMetaData"
+            if path == "":
+                path = "*"
+            req["path"] = path
+
         return self._sendReceiveMsg(req, timeout)
 
     # Set value to a given path
     def setValue(self, path, value, attribute="value", timeout=5):
         if 'nan' == value:
             return json.dumps({"error": path + " has an invalid value " + str(value)}, indent=2)
         req = {}
-        req["action"] = "set"
-        req["path"] = path
-        req["attribute"] = attribute
+        if self.subprotocol == "VISSv2":
```

Comment:
Standard VISSv2 only supports setting target value and reading current value, so I suggest we give an error if someone use "setValue" and instead support "setTargetValue". Similar give an error if someone call "getTargetValue"

Reply:
Pushed an update to this effect.